src/org-binder/org-tasks-provider.ts

```diff
@@ -320,31 +325,25 @@ export class TaskProvider {
                         }
                     }
                     task.result = await that.partitionWriter.createOrganizationalUnit(resource, parentId);
-                    that.state.setBinding({
-                        type: resource.type,
-                        logicalId: resource.logicalId,
-                        lastCommittedHash: hash,
-                        physicalId: createOrganizationalUnitTask.result,
-                        partitionId: task.result,
-                    });
+                    that.state.setBindingHash( resource.type, resource.logicalId, hash);
```

Comment:
@brcourt I noticed "main" uses setBindingHash, it seems you changed this to setBinding. I changed it back because it was this change overwrites the physicalIds of the OUs. what was the reason for the change? does changing this back have impact on govcloud support?

Reply:
Actually, main is using `setBindingPhysicalId` in this spot. I switched it to just `setBinding` because 1. I don't see the point of having a function that only manages physicalIds in bindings, when the `setBinding` function can do the same thing, and 2. the `setBindingPhysicalId` function updates the physicalId when it is called and the binding exists (which is a problem when I want to update the partitionId and not physicalId). Using `setBinding` just made more sense to me.